README.md

```diff
@@ -25,35 +25,38 @@ Any contribution that can help improve this list and make it more comprehensive
 
 ## Table of Contents
 
-- [Basics](#basics)
```

Comment:
This was changed by my vscode. If you prefer not to include `Contribution` and `TOC`, I can change them back.